apps/credential-showcase-api-server/src/database/repositories/ScenarioRepository.ts

```diff
@@ -614,6 +632,7 @@ class ScenarioRepository implements RepositoryDefinition<Scenario, NewScenario>
       const actionsResult = await tx
         .insert(stepActions)
         .values(
+          // @ts-ignore
```

Comment:
why do we need to ignore?

Reply:
The values() function does not accept undefined and the compiler complains. However you just made me realize that there was an issue in my code.